charts/k8ssandra/values.yaml

```diff
@@ -464,7 +464,7 @@ medusa:
     # -- Specifies the container repository for Medusa
     repository: docker.io/k8ssandra/medusa
     # -- Tag of an image within the specified repository
-    tag: 0.10.1
+    tag: 5ba9003
```

Comment:
We should probably cut out a Medusa release so that we can use a stable tag.

Reply:
I've released Medusa 0.11.0 with the changes you need for this PR.

Reply:
Done in b6daeae.